app/Http/Controllers/SaveResourceController.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Resources\SaveResourceDescriptorResource;
+use App\Models\Save;
+use App\Models\SaveResource;
+use App\Services\SaveResourceService;
+use Illuminate\Http\Request;
+use Illuminate\Http\Resources\Json\AnonymousResourceCollection;
+use Illuminate\Validation\Rules\File;
+use Response;
+
+class SaveResourceController extends Controller
+{
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param \Illuminate\Http\Request $request
+     * @return AnonymousResourceCollection
+     */
+    public function store(SaveResourceService $saveResourceService, Save $save, Request $request)
+    {
+        $this->authorize("update", $save);
+        $request->validate([
+            "resources.*" => [
+                File::types(SaveController::ALLOWED_MIMETYPES)
+                    ->max(SaveController::FILE_MAX_KILOBYTES)
+            ]
+        ]);
+        return SaveResourceDescriptorResource::collection($saveResourceService->saveResources($save, $request->allFiles()));
+
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param \App\Models\SaveResource $resource
+     * @return \Illuminate\Http\Response
+     */
+    public function show(SaveResource $resource)
```

Comment:
This can later removed if the double primary keys are added and the ID removed. 

For more info on this issue see: https://trello.com/c/Bwu2JG1K/458-ressourcenport-im-backend-hinzuf%C3%BCgen#comment-651f13a7c290023389d18a96